xml/chapter5/section5/section5.xml

```diff
@@ -440,11 +586,16 @@ assign("fun", list(op("lookup_symbol_value"), constant("f"), reg("env")))
 
   <TEXT>
     A compiler can also optimize access to the environment.  Having
-    analyzed the code, the compiler can in many cases know in which frame
```

Comment:
In the JavaScript edition, this is "always" not just "in many cases" because of scanning out names, even at top level.